src/emmc.c

```diff
@@ -771,6 +771,11 @@ pu_emmc_parse_partitions(PuEmmc *emmc,
 
     emmc->partitions = g_list_reverse(emmc->partitions);
 
+    /* Reserve 34 blocks for secondary GPT */
+    if (g_str_equal(emmc->disktype->name, "gpt")) {
+        fixed_parts_size += 34;
```

Comment:
Do you know where this size is specified and its reasoning? These 34 sectors seem to be reserved for both the primary and secondary partition table (regardless of GPT or MS-DOS) by most partitioning tools (like fdisk and sgdisk). The MBR takes up 512 bytes, which would be one sector, so I am wondering what the other 33 sectors are for.

We may want to use a define for this size.

Reply:
The other sectors are used for GPT. Sector 0 is the MBR, sector 1 the primary GPT header and sectors 2-33 contain information on the partitions. Each 512 byte sector describes 4 partitions for 128 partitions in 32 sectors.
GPT also uses the last 33 sectors to store the same information for redundancy in case the primary GPT gets corrupted. See the graphic in https://en.wikipedia.org/wiki/GUID_Partition_Table. 
I reserve 34 sectors to not have an uneven number of usable sectors.